### DISCUSSION TITLE
Consider removing inline keyword for member function

### DISCUSSION THREAD

**Discussion by Paraworker** [CuarzoSoftware/Louvre#48] [Ideas]
The meaning of the inline keyword has changed for a long time, and whether a function is inlined or not is entirely determined by the compiler.

**Comment by ehopperdietzel** [comment]
Yep, thank you, I'm not sure if that applies to all compilers and settings, so I'll conduct some benchmarks before making a decision.

**Reply by Paraworker** [reply]
Sorry for my delayed response. According to the cppreference:

> A function defined entirely inside a class/struct/union definition, whether it's a member function or a non-member friend function, is implicitly an inline function unless it is attached to a named module(since C++20).

So we don't need to mark it explicitly anyway.

**Reply by ehopperdietzel** [reply]
Okay, I'll remove it.
